api/src/main/resources/workbench-api.yaml

```diff
@@ -5665,12 +5736,12 @@ definitions:
   UserListResponse:
```

Comment:
Maybe `AdminTableUserListResponse`? There will likely be other places we list users.